tools/esdlcmd/esdl-publish.cpp

```diff
@@ -94,8 +93,6 @@ class EsdlPublishCmdCommon : public EsdlCmdCommon
             return true;
         if (iter.matchFlag(optWSProcPort, ESDL_OPTION_SERVICE_PORT) || iter.matchFlag(optWSProcPort, ESDL_OPT_SERVICE_PORT))
             return true;
-        if (iter.matchFlag(optVersionStr, ESDLOPT_VERSION))
```

Comment:
I may also refactor the variable name for the definition sequence. What better describes this variable, "definitionVersion" or "defintionSequence", @rpastrana @afishbeck ?